snaps/build-and-release-k8s-snaps.sh

```diff
@@ -4,6 +4,7 @@ set -eux
 
 KUBE_VERSION="${KUBE_VERSION:-$(curl -L https://dl.k8s.io/release/stable.txt)}"
 KUBE_ARCH="amd64"
+ADDONS_BRANCH_VERSION="release-${KUBE_VERSION:1:3}"
```

Comment:
@ktsakalozos Now that https://github.com/juju-solutions/kubernetes-jenkins/pull/108 has landed, can you fix this to use `get_major_minor` as well?